core_v2/sources/domains.move

```diff
@@ -60,6 +60,34 @@ module aptos_names_v2::domains {
         capability: SignerCapability,
     }
 
+    /// Manager object refs
+    struct Manager has key {
+        /// The extend_ref of the manager object to get its signer
+        extend_ref: object::ExtendRef,
+    }
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct DomainNameRecordTokenV2 has key {
+        name_domain_name: String,
+        expiration_time_sec: u64,
+        target_address: Option<address>,
+
+        extend_ref: object::ExtendRef,
+        transfer_ref: object::TransferRef,
+    }
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct SubdomainNameRecordTokenV2 has key {
+        name_subdomain_name: String,
+        expiration_time_sec: u64,
+        target_address: Option<address>,
+        domain: Object<DomainNameRecordTokenV2>,
```

Comment:
I can think of 2:
1. get the full name of the subdomain -> we need the domain name
2. get the expiration date when it's auto-renew-> we need the domain object

Reply:
```
    inline fun subdomain_token_addr_inline(
        domain_token: Object<DomainNameRecordTokenV2>,
        subdomain_name: String,
    ): address acquires DomainNameRecordTokenV2 {
        let domain_token_addr = object::object_address(&domain_token);
        let subdomain_collection_name = &borrow_global<DomainNameRecordTokenV2>(domain_token_addr).domain_name;
        token::create_token_address(&domain_token_addr, subdomain_collection_name, &subdomain_name)
    }
```
it's used to get the subdomain token address since it's the creator of the subdomain collection